### PR TITLE
add panedr package

### PR DESCRIPTION
**Pull request by kain88-de**
ping @jbarnoud to be a maintainer as well.

This is a nice package to read data files from MD simulations with GROMACS

### CODE REVIEW

**Comment by kain88-de** [comment]
appveyor failures on 3.5 seem unrelated to the package itself.

**Comment by dougalsutherland** [comment]
Yeah, those are temporarily broken: see https://github.com/conda-forge/idna-feedstock/pull/5.

File: recipes/panedr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - python
+    - pandas
+
+test:
+  imports:
+    - panedr
+
+about:
+  home: http://github.com/jbarnoud/panedr
+  license: LGPL
```
Comment (synapticarbors) [inline recipes/panedr/meta.yaml]: `license: LGPL-2.1`

File: recipes/panedr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
+  summary: 'GROMACS edr files to pandas dataframe'
+  # The remaining entries in this section are optional, but recommended
```
Comment (synapticarbors) [inline recipes/panedr/meta.yaml]: Delete comment.

**Comment by jbarnoud** [comment]
Thanks @kain88-de !

**Comment by kain88-de** [comment]
@synapticarbors is there anything else blocking this package?

**Comment by synapticarbors** [comment]
I think if you close and re-open this same PR then the `idna` issue should be fixed. Once appveyor runs and everything looks good, I'll merge.

**Pull request closed without merging**